exasol_transformers_extension/udfs/models/named_entity_recognition_udf.py

```diff
@@ -0,0 +1,183 @@
+import torch
+import pandas as pd
+import transformers
+from typing import Tuple, List, Union, Dict
+from exasol_transformers_extension.deployment import constants
+from exasol_transformers_extension.utils import device_management, \
+    dataframe_operations, bucketfs_operations
+
+
+class NamedEntityRecognitionUDF:
```

Comment:
Is this only NER or also TokenClassification

Reply:
This is only NER